scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -2,28 +2,140 @@
 
 import javax.sql.DataSource;
 
+import java.sql.Connection;
+import java.sql.SQLException;
 import java.util.Optional;
 
 import gov.nasa.jpl.aerie.scheduler.server.ResultsProtocol;
+import gov.nasa.jpl.aerie.scheduler.server.exceptions.NoSuchRequestException;
+import gov.nasa.jpl.aerie.scheduler.server.exceptions.NoSuchSpecificationException;
 import gov.nasa.jpl.aerie.scheduler.server.models.SpecificationId;
 import gov.nasa.jpl.aerie.scheduler.server.remotes.ResultsCellRepository;
+import gov.nasa.jpl.aerie.scheduler.server.services.ScheduleResults;
+import org.apache.commons.lang3.NotImplementedException;
 
 public record PostgresResultsCellRepository(DataSource dataSource) implements ResultsCellRepository {
```

Comment:
When I initially stubbed `PostgresResultsCellRepository` I made it a `record` to save myself some boilerplate. We don't need to expose `dataSource()` publicly, so if you wanted to you could make this a `class` with a `private final Datasource dataSource`. Thought it was worth mentioning, no obligation to change this.

Reply:
Good to know, don't resolve this comment until next week so that I hopefully see it again then, otherwise I'll surely forget.

Reply:
For sure, not a big deal if it's not changed.

Reply:
This should be addressed now too. I made it a record to keep that variable private!

Reply:
Taken care of!